package.json

```diff
@@ -163,10 +161,12 @@
     "lodash.throttle": "4.1.1",
     "mixpanel": "0.10.2",
     "moment": "2.23.0",
+    "node-polyfill-webpack-plugin": "^1.1.4",
     "prop-types": "15.6.2",
     "punycode": "^2.1.1",
     "qrcode.react": "0.8.0",
     "react": "16.6.3",
+    "react-16-node-hanging-test-fix": "^1.0.0",
```

Comment:
Not even kidding, that's the official fix until the next React version is released (and this package is already a year old).

And yeah, can move to devDeps since it's only for tests.